src/workloads/query/ShardFilter.yml

```diff
@@ -0,0 +1,101 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of queries which need to perform shard filtering.
+Keywords:
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  Namespace: &Namespace test.Collection0
+  MaxPhases: &MaxPhases 8
+  DocumentCount: &DocumentCount 1000
```

Comment:
Why do we want such a small collection?

Reply:
It matters because of what we're measuring. Because we don't have optimizer metrics yet, we only get stats about client reported metrics (overall query latency and throughput basically). This means that depending on what size of collection you pick, different results will be visible in the dashboard (i.e. its possible that either a slow execution time or a slow planning time will be masked by something else).

Small collection = planning time will probably dominate based on the other workloads I've looked at for Bonsai, while a larger collection = execution time will probably dominate. If you want to focus on one or the other, that can inform your choice of cardinality- for instance, we recently added a small-collection workload with large queries designed to stress the planner. If you want both data points, I would recommend having a "small" and "large" version of each query.

From a shipping M2 perspective, I'd say both is desirable.